test_inflection.py

```diff
@@ -327,8 +327,8 @@ def test_singularize_plural(singular, plural):
 
 @pytest.mark.parametrize(("singular", "plural"), SINGULAR_TO_PLURAL)
 def test_pluralize_plural(singular, plural):
-    assert plural == inflection.pluralize(plural)
-    assert plural.capitalize() == inflection.pluralize(plural.capitalize())
+    assert plural == inflection.pluralize(singular)
```

Comment:
> fix a test that looks like it had a copy/paste mistake.

Actually, this is not a copy-paste mistake. We're testing here that pluralizing something that's already in plural does n't pluralize it again :)